app/app.js

```diff
@@ -155,18 +155,18 @@ module.exports = async (options) => {
   })
 
   // Component example preview
-  app.get('/components/:componentName/:exampleName*?/preview', function (req, res, next) {
+  app.get('/components/:componentName/:exampleName?/preview', function (req, res, next) {
     // Find the data for the specified example (or the default example)
     const componentName = req.params.componentName
     const exampleName = req.params.exampleName || 'default'
 
-    const previewLayout = res.locals.componentData.previewLayout || 'layout'
+    const previewLayout = res.locals.componentData?.previewLayout || 'layout'
 
-    const exampleConfig = res.locals.componentData.examples.find(
+    const exampleConfig = res.locals.componentData?.examples.find(
       example => example.name.replace(/ /g, '-') === exampleName
     )
 
-    if (!exampleConfig) {
+    if (!exampleConfig || !componentName) {
```

Comment:
We hadn't checked if `componentName` was found

Reply:
Thanks for pointing this one out, must have been left over from before the route tweak:
https://github.com/alphagov/govuk-frontend/pull/2987/files#r1039983633